src/postgres/src/test/regress/expected/yb_float8.out

```diff
@@ -556,4 +556,3 @@ FROM (SELECT 10*cosd(a), 10*sind(a)
 (5 rows)
 
 RESET extra_float_digits;
-DROP TABLE FLOAT8_TBL;
```

Comment:
Note: although this reduces diff with the original test, it might make our existing tests fail.  We should keep this change until some obvious error shows up like a test depending on that table existing.